<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1151</width>
      <height>557</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="pessoasTtitle">
           <property name="text">
            <string>Pessoas</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="nomePessoalbl">
             <property name="text">
              <string>Nome:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nomePessoaTxt">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="idadeLbl">
             <property name="text">
              <string>Idade:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="idadeTxt">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="generoLbl">
             <property name="text">
              <string>Gênero:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="generoCbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="enderecoLbl">
             <property name="text">
              <string>Endereço:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enderecoTxt">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="telefoneLbl">
             <property name="text">
              <string>Telefone:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="telefoneTxt">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="emailLbl">
             <property name="text">
              <string>E-mail:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailTxt">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_41">
           <item>
            <widget class="QLabel" name="tipoLbl">
             <property name="text">
              <string>Tipo Pessoa:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="tipoCbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="medicosLbl">
             <property name="text">
              <string>Médicos:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="medicosTxt"/>
           </item>
           <item>
            <widget class="QLabel" name="historicoLbl">
             <property name="text">
              <string>Historico:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="historicoTxt"/>
           </item>
           <item>
            <widget class="QPushButton" name="testeBtn">
             <property name="text">
              <string>Atualizar Memória</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_37">
             <item>
              <widget class="QPushButton" name="historicoBtn">
               <property name="text">
                <string>Histórico</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="criarPessoaBtn">
               <property name="text">
                <string>Criar Pessoa</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_22">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_23">
         <item>
          <widget class="QLabel" name="exameTitle">
           <property name="text">
            <string>Exame</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_27">
           <item>
            <widget class="QLabel" name="dataLbl">
             <property name="text">
              <string>Data:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dataTxt">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_39">
           <item>
            <widget class="QLabel" name="pacienteExameLbl">
             <property name="text">
              <string>Paciente:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pacienteExameCbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_28">
           <item>
            <widget class="QLabel" name="medicoExameLbl">
             <property name="text">
              <string>Médico:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="medicoExameCbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QLabel" name="especialidadeLbl">
             <property name="text">
              <string>Especialidade:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="especialidadeCbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <widget class="QLabel" name="enderecoExameLbl">
             <property name="text">
              <string>Endereço:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="enderecoExameTxt">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_31">
           <item>
            <widget class="QLabel" name="resultadoLbl">
             <property name="text">
              <string>Resultado:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="resultadoCbox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_24">
           <item>
            <widget class="QLabel" name="examesLbl">
             <property name="text">
              <string>Exames:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="examesTxt"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_38">
             <item>
              <widget class="QPushButton" name="criarExameBtn">
               <property name="text">
                <string>Criar Exame</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="processarExame">
               <property name="text">
                <string>Processar Exame</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="pessoasTtitle_2">
         <property name="text">
          <string>Consulta</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="pacienteConsultaLbl">
           <property name="text">
            <string>Paciente:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="pacienteConsultaCbox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="medicoConsultaLbl">
           <property name="text">
            <string>Medico:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="medicoConsultaCbox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="exameConsultaLbl">
           <property name="text">
            <string>Exame:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="adicionarExameBtn">
           <property name="text">
            <string>Adicionar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="exameConsultaCbox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="consultorioLbl">
           <property name="text">
            <string>Consultório:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="consultorioCbox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QLabel" name="resumoConsultaLbl">
           <property name="text">
            <string>Resumo Consulta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="resumoConsultaTxt"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_40">
           <item>
            <widget class="QPushButton" name="letExamesBtn">
             <property name="text">
              <string>Ver Resumo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="marcarConsultaBtn">
             <property name="text">
              <string>Marcar Consulta</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="pessoasTtitle_3">
         <property name="text">
          <string>Marcações</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_15">
         <item>
          <widget class="QLabel" name="alteracoesConsulta">
           <property name="text">
            <string>Alterações Consulta:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="alteracoesConsultaTxt"/>
         </item>
         <item>
          <widget class="QLabel" name="resultadoExamesLbl">
           <property name="text">
            <string>Exames:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="resultadoExamesTxt"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_44">
           <item>
            <widget class="QPushButton" name="lerConsultasBtn">
             <property name="text">
              <string>Ler Consultas</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="enviarAlertaBtn">
             <property name="text">
              <string>Enviar Alerta</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1169</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
